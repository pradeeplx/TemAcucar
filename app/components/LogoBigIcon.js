import React from 'react'
import { Image, PixelRatio } from 'react-native'
import { maxLower } from '../helpers'

export default LogoBigIcon = ({ style }) => {
  const ratio = PixelRatio.get()
  const result = maxLower(ratio, 1, 1.5, 2, 3, 3.5, 4)
  const image = require(`../img/bigIcon${result * 100}.png`)
  return (
    <Image source={image} style={[{
      width: 40,
      height: 40,
    }, style]}/>
  )
}
