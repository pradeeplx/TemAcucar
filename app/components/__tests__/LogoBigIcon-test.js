import 'react-native'
import React from 'react'
import LogoBigIcon from '../LogoBigIcon'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <LogoBigIcon
      style={{top: 0}}
    >
      FooBar
    </LogoBigIcon>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
