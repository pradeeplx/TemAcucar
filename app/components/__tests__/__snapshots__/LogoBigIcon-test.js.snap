exports[`test renders correctly 1`] = `
<Image
  source={1}
  style={
    Array [
      Object {
        "height": 40,
        "width": 40,
      },
      Object {
        "top": 0,
      },
    ]
  } />
`;
